apps/website-3/src/app/[locale]/error.tsx

```
@@ -0,0 +1,39 @@
+'use client';
+
+import {useTranslations} from 'next-intl';
+import {useEffect} from 'react';
+
+type Props = {
+  error: Error;
+  reset(): void;
+};
+
+export default function Error({error, reset}: Props) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)